### ISSUE TITLE
Default email to site admin

### ISSUE THREAD

**Issue by brashrebel** [brashrebel/don8#9]
Not sure if it is a great idea but could be cool to set the email option to default to the site's admin email address.

**Comment by burhandodhy** [comment]
Is this still open ? I'm very interested in doing this

**Comment by brashrebel** [comment]
@burhandodhy yeah, I think so! It has been three years since I created the issue but I think it still applies.

**Comment by burhandodhy** [comment]
Good, I'm going to start work on this ;)
